pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/readers/forward/VarByteChunkMVForwardIndexReader.java

```diff
@@ -179,6 +183,20 @@ private byte[] getBytesCompressed(int docId, ChunkReaderContext context) {
     return bytes;
   }
 
+  public void recordDocIdRanges(int docId, List<ValueRange> ranges) {
```

Comment:
this is a private method? 

btw, changes in this class look quite similar to those in VarByteChunkSVForwardIndexReader.java below, so is there chance to extract common parts to reuse?